MediaBrowser.Providers/Tmdb/Movies/TmdbSearch.cs

```diff
@@ -18,8 +19,22 @@ namespace MediaBrowser.Providers.Tmdb.Movies
 {
     public class TmdbSearch
     {
-        private static readonly CultureInfo EnUs = new CultureInfo("en-US");
-        private const string Search3 = TmdbUtils.BaseTmdbApiUrl + @"3/search/{3}?api_key={1}&query={0}&language={2}";
+        private static readonly CultureInfo _usCulture = new CultureInfo("en-US");
+
+        private static readonly Regex _cleanEnclosed = new Regex(@"\p{Ps}.*\p{Pe}", RegexOptions.Compiled);
+        private static readonly Regex _cleanNonWord = new Regex(@"[\W_]+", RegexOptions.Compiled);
+        private static readonly Regex _cleanStopWords = new Regex(@"\b( # Start at word boundary
+            19[0-9]{2}|20[0-9]{2}| # 1900-2099
+            S[0-9]{2}| # Season
+            E[0-9]{2}| # Episode
+            (2160|1080|720|576|480)[ip]?| # Resolution
+            [xh]?264| # Encoding
```

Comment:
there's waaaaaaaaaaaaaaaaaaaay more encodings in there, at least `avc`, `hevc`, `h265` (plus maybe `divx`, `xvid` and whatnot else, but those 3 should be the most popular)

Reply:
so at least make something like `([xh]?26[45]|avc|hevc)|` here

Reply:
i think we could include 1440 res, and vp8, vp9, av1 as well.